en/material-system/pass-parameter-list.md

```diff
@@ -43,7 +45,17 @@ This property doesn't exist by default, which means the pass is executed uncondi
 Specifies the rendering priority of the current pass, the bigger the number, the lower the priority. The default is (128), min is (0), max is (255), arithmetic operations between these constants and integer constants are supported.
 
 ## Stage
-Specifies which render stage the current pass belongs to.
+Specifies which render stage the current pass belongs to. For the built-in forward pipeline, the only available stage is 'default'.
+
+## Phase
+Specifies which phase the current pass belongs to. For the built-in forward pipeline, the available phases are 'default', 'forward-add' and 'shadow-caster'.
+
+## PropertyIndex
+Specifies the index of the pass to copy runtime property data from. Could be useful in some cases, e.g. the forward-add pass needs the same uniform data as the base pass.<br>
```

Comment:
Specifies the index of the pass to copy runtime property data from. Could be useful in some cases, e.g. the forward-add pass needs the same uniform data as the base pass.<br>
Once specified, all the properties for the current pass will not be visible on the material inspector.
-->
Specifies the index of the pass to copy runtime property data from.  This could be useful in some cases, e.g. the forward-add pass needs the same uniform data as the base pass. Once specified, all the properties for the current pass will not be visible in the material inspector.

Reply:
Specifies the index of the pass to copy runtime property data from.

This line doesn't make sense to me...can you clarify?

Reply:
Yeah, I was hesitant about this one too, it's hard to clarify in a single sentence. When two passes need to share the same set of properties, `propertyIndex` can be specified to avoid the need for artists to specify that same set of data multiple times. The index specified will be used at runtime to copy the properties in the target pass into the current pass.